### ISSUE TITLE
sanitizeDerivationName seems unnecessary

### ISSUE THREAD

**Issue by wmertens** [nix-community/dream2nix#260]
there's
```nix
{
  sanitizeDerivationName = name:
    lib.replaceStrings ["@" "/"] ["__at__" "__slash__"] name;
}
```

which results in long and unreadable names for modules like `__at__babel__slash__runtime`, and there's no technical need for it, since Nix will sanitize names like `@babel/runtime` to `-babel-runtime`.

How about not sanitizing derivation names?

**Comment by tinybeachthor** [comment]
I think this makes sense. 
I prefer the shorter name, and there is no real information lost between `__at__babel__slash__runtime` and `-babel-runtime`.

Could this change break anything though?

**Comment by wmertens** [comment]
No breaking change, it only means rebuilding packages

**Comment by DavHau** [comment]
I'm not opposed to removing the custom sanitizing. I just thought, that with `-babel-runtime` it might not be clear on first glance if this is a namespaced package or a package called `babel-runtime`, but actually the `-` indicates a namespace.
But if we had `@babel/runtime-foo` it would be rendered to `-babel-runtime-foo`, making it unclear if `runtime` is part of the namespace or package name.

**Comment by DavHau** [comment]
OK, I'm fine with removing the use of sanitizDerivationName.

**Comment by tinybeachthor** [comment]
Resolved in #315
